tmexp/preprocess.py

```diff
@@ -147,13 +183,19 @@ def preprocess(
 
     files_content: Dict[str, Dict[str, Any]] = defaultdict(dict)
     sql = FILE_CONTENT % (repo, ",".join("'%s'" % ref for ref in refs), languages)
-    uast_xpath = " | ".join([FEATURE_MAPPING[feature]["xpath"] for feature in features])
     if stem:
         stemmer = PorterStemmer()
     vocabulary: Dict[str, Set[str]] = {feature: set() for feature in features}
     client = bblfsh.BblfshClient(bblfsh_host + ":" + str(bblfsh_port))
-    lang_count = Counter()
+    parsed_count: Counter = Counter()
```

Comment:
Should be CounterType with its type argument

Reply:
done